scripts/pre_commit_linter.py

```diff
@@ -964,8 +964,11 @@ def _check_comments(all_files):
             for line_num in range(file_length):
                 line = file_content[line_num].lstrip().rstrip()
                 next_line = ''
+                previous_line = ''
                 if line_num + 1 < file_length:
                     next_line = file_content[line_num + 1].lstrip().rstrip()
+                if -1 < line_num - 1 < file_length:
```

Comment:
I think the condition `line_num > 0` would suffice.

Reply:
Done.